common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/ElasticSearchClient.scala

```diff
@@ -46,9 +46,10 @@ trait ElasticSearchClient extends ElasticSearchExecutions with GridLogging {
   //TODO: this function should fail and cause healthcheck fails
   def ensureAliasAssigned() {
     logger.info(s"Checking alias $imagesCurrentAlias is assigned to index…")
-    if (getCurrentAlias.isEmpty) {
+    val indexForCurrentAlias = Await.result(getIndexForAlias(imagesCurrentAlias), tenSeconds)
+    if (indexForCurrentAlias.isEmpty) {
       ensureIndexExists(initialImagesIndex)
```

Comment:
This might read slightly better if we renamed `ensureIndexExists` to `createIndexIfMissing`

Reply:
yeah good idea, will do that now